firefox-ios/firefox-ios-tests/Tests/UnitTest.xctestplan

```diff
@@ -217,6 +216,7 @@
         "TabToolbarHelperTests\/testTelemetryForSiteMenu()",
         "TelemetryWrapperTests\/test_error_cpuExceptionIsCalled()",
         "TelemetryWrapperTests\/test_loginWithEmail_GleanIsCalled()",
+        "TermsOfServiceTelemetryTests",
```

Comment:
remove? I think we added this back in here: https://github.com/mozilla-mobile/firefox-ios/pull/30033